ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+---
+# tasks file for cyhy_feeds
+- name: Grab GitHub OAuth token from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: github_oauth_token.yml
+    dest: /tmp/github_oauth_token.yml
+    mode: get
+  become: no
```

Comment:
Throughout this file you have `become: no`, but these lines are not needed since `become` defaults to `no`.

Reply:
The "become: no" is needed because it is a local_action that will want to run as sudo requiring a password